frontend/src/component/feature/StrategyTypes/NewStrategyVariants.tsx

```diff
@@ -123,10 +118,10 @@ export const NewStrategyVariants: FC<{
 
     return (
         <>
-            <StyledVariantsHeader>
+            <Alert severity='info' sx={{ mb: 2 }} icon={false}>
```

Comment:
Technically, shouldn't it be up to the containing parent to decide on spacing between this and the next item? But it's not important. Up to you 🤷🏼